spec/models/movie_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Movie, type: :model do
+  before do
+    create(:movie)
+  end
+
+  context 'when creating new hall' do
```

Comment:
```suggestion
  context 'when creating new movie' do
```

Reply:
Cmd+c Cmd+v :) Thanks for pointing that out